subsys/bluetooth/host/hci_core.c

```diff
@@ -4405,7 +4433,7 @@ static int read_ext_features(void)
 		cp = net_buf_add(buf, sizeof(*cp));
 		cp->page = i;
 
-		err = bt_hci_cmd_send_sync(BT_HCI_OP_READ_LOCAL_EXT_FEATURES,
+		err = hci_cmd_send_sync(BT_HCI_OP_READ_LOCAL_EXT_FEATURES,
 					   buf, &rsp);
```

Comment:
Seems like the alignment with `(` will be off since you're not adjusting the second line after making the first shorter. I won't comment on all similar places in this patch, but it seems there are several of them.